src/depgraphs.nim

```diff
@@ -478,9 +492,17 @@ proc debugFormular(c: var AtlasContext; g: var DepGraph; f: Form; s: Solution) =
 proc solve*(c: var AtlasContext; g: var DepGraph; f: Form) =
   let m = f.idgen
   var s = createSolution(m)
-  #debugFormular c, g, f, s
 
-  if satisfiable(f.f, s):
+  var status =
+    try: satisfiable(f.f, s)
+    except SatOverflowError as err:
+      echo "\n"
+      echo "SAT MaxIterationLimitError: "
+      debugFormular c, g, f, s
+      echo "\n"
```

Comment:
Other parts of the codebase don't use `echo` to report errors.

Reply:
Good catch, just left over debugging statements. I'll go through and clean it up.